### PR TITLE
adjust GE options for WSV

### LINKED ISSUE #369: Guided Experience: Selecting veteran + WWII should hide the still-serving option
This logic is already implemented in the RadioSelector component. Is it possible to re-use this?

### PR DESCRIPTION
**Pull request by sastels**
fixes #369 

for WSV:
* hide "still serving" option
* if patronType === Veteran, skip statusAndVitals question entirely (only 1 valid option, "released")

### CODE REVIEW

**Comment by maxneuvians** [comment]
Two small issues:

* Click Veterans or Serving
* Click WW2
= Back button is broken

* Click Veterans or Serving
* Click CAF
* Click Still serving
* Click back
* Click WW2
= Can no longer change Still Serving

Both are not big problems, especially the second one is very conditional.

**Comment by sastels** [comment]
👍  good catch!

**Review by maxneuvians — APPROVED** [review]
🦑 LGTM

File: locales/en/common.json [hunk 1]
```diff
@@ -114,8 +114,6 @@
   "service-person": "Veteran or serving member",
   "family": "Family members",
   "organization": "Organizations",
-  "released": "Released",
-  "still serving": "Still Serving",
```
Comment (smcmurtry) [inline locales/en/common.json:118]: Thanks for cleaning up some of the unused variables - I'm sure there are more. We should probably make an issue to clean it up and put everything in sections.

**Review by smcmurtry — APPROVED** [review]
Looks good!

File: locales/en/common.json (hunk 1 above)
Comment (sastels) [reply locales/en/common.json:118]: Probably, though better might be to move most of this text to AirTable (though we'd still have to figure out what's still needed). putting things in sections would probably help, mind you.

**Pull request merged**